### ISSUE TITLE
Unclear docs/handling for :linters :unresolved-symbol :exclude

### ISSUE THREAD

**Issue by Cnly** [clj-kondo/clj-kondo#1509]
**version**

clj-kondo v2021.12.01

**macro usage**

Related to configuration for custom macros

**platform**

Mac, binary

**editor**

N/A

**problem**

clj-kondo has [support](https://github.com/clj-kondo/clj-kondo/blob/master/doc/linters.md#unresolved-symbol) for excluding unresolved symbols in custom macros. However the docs didn't make clear that

* macros need to be supplied as fq-names
* There can't be more than one macro in a `()` list

which are a bit unintuitive IMO.

**repro**

```clj
;; a.clj
(ns ^{:clj-kondo/config '{:linters
                          {:unresolved-symbol
                           {:exclude [(a/my-macro1 a/my-macro2 [$])]}}}}  ;; "a/my-macro2" here is unwanted
  a)

(my-macro1 $ ? !)
```

Output:

```console
$ clj-kondo --lint a.clj
Unexpected error. Please report an issue.
java.lang.IllegalArgumentException: Don't know how to create ISeq from: clojure.lang.Symbol
        at clojure.lang.RT.seqFrom(RT.java:557)
        at clojure.lang.RT.seq(RT.java:537)
        at clojure.core$seq__5420.invokeStatic(core.clj:139)
        at clojure.core$reduce1.invokeStatic(core.clj:932)
        at clojure.core$set.invokeStatic(core.clj:4101)
        at clj_kondo.impl.config$fn__3049$delayed_cfg__3050$fn__3052$fn__3056.invoke(config.clj:225)
        at clojure.core$update.invokeStatic(core.clj:6185)
        at clj_kondo.impl.config$fn__3049$delayed_cfg__3050$fn__3052.invoke(config.clj:219)
        at clojure.lang.ArrayChunk.reduce(ArrayChunk.java:63)
        at clojure.core.protocols$fn__8178.invokeStatic(protocols.clj:136)
        at clojure.core.protocols$fn__8178.invoke(protocols.clj:124)
        at clojure.core.protocols$fn__8138$G__8133__8147.invoke(protocols.clj:19)
        at clojure.core.protocols$seq_reduce.invokeStatic(protocols.clj:31)
        at clojure.core.protocols$fn__8170.invokeStatic(protocols.clj:75)
        at clojure.core.protocols$fn__8170.invoke(protocols.clj:75)
        at clojure.core.protocols$fn__8112$G__8107__8125.invoke(protocols.clj:13)
        at clojure.core$reduce.invokeStatic(core.clj:6833)
        at clj_kondo.impl.config$fn__3049$delayed_cfg__3050.invoke(config.clj:213)
        at clojure.lang.AFn.applyToHelper(AFn.java:154)
        at clojure.lang.AFn.applyTo(AFn.java:144)
        at clojure.core$apply.invokeStatic(core.clj:667)
        at clojure.core$memoize$fn__6895.doInvoke(core.clj:6342)
        at clojure.lang.RestFn.invoke(RestFn.java:408)
        at clj_kondo.impl.config$fn__3049$fn__3060.invoke(config.clj:236)
        at clj_kondo.impl.namespace$reg_unresolved_symbol_BANG_.invokeStatic(namespace.clj:283)
        at clj_kondo.impl.linters$lint_var_usage.invokeStatic(linters.clj:283)
        at clj_kondo.core$run_BANG_.invokeStatic(core.clj:173)
        at clj_kondo.main$main.invokeStatic(main.clj:130)
        at clj_kondo.main$main.doInvoke(main.clj:126)
        at clojure.lang.RestFn.applyTo(RestFn.java:137)
        at clojure.core$apply.invokeStatic(core.clj:667)
        at clj_kondo.main$_main$fn__16515.invoke(main.clj:177)
        at clj_kondo.main$_main.invokeStatic(main.clj:177)
        at clj_kondo.main$_main.doInvoke(main.clj:175)
        at clojure.lang.RestFn.applyTo(RestFn.java:137)
        at clj_kondo.main.main(Unknown Source)
```

**config**

N/A

**expected behavior**

I'd expect either the docs make it clear with more examples or the code accepts some additional intuitive usages. Either way if there're not a ton of changes I can make a PR. :)

**Comment by borkdude** [comment]
The intention of `(foo.bar/baz [x])` is: in calls to `foo.bar/baz`, ignore `x`. Var symbols in clj-kondo config should always be fully qualified. PR welcome.

**Comment by borkdude** [comment]
We now also have a linter for `:clj-kondo-config` which we can add this to. That would be even better than docs.
